python-libraries/nanover-openmm/src/nanover/openmm/imd.py

```diff
@@ -357,3 +362,13 @@ def get_imd_forces_from_system(system: app.Simulation) -> List[mm.CustomExternal
         and force.getEnergyFunction() == IMD_FORCE_EXPRESSION
         and force.getNumParticles() == system_num_particles
     ]
+
+
+def get_sparse_forces(forces: npt.NDArray) -> Tuple[npt.NDArray, npt.NDArray]:
```

Comment:
Could do with a docstring here to be explicit about the meaning of sparse forces i.e that it's a pair of lists of forces and the corresponding particle index (I think?)

Reply:
I have since added a docstring to this function, let me know if it needs further editing!